core/docker/Dockerfile

```diff
@@ -45,7 +45,8 @@ RUN ${BUILD_DIR}/scripts/generate_datafed.sh && \
 		-DBUILD_WEB_SERVER=False		\
 		-DBUILD_DOCS=False					\
 		-DBUILD_PYTHON_CLIENT=False	\
-		-DBUILD_FOXX=False
+		-DBUILD_FOXX=False \
+        -DCMAKE_BUILD_TYPE=Debug
```

Comment:
I don't think you meant to commit this. We don't want the images built with Debug code. That was probably during testing.